src/mon/Monitor.h

```diff
@@ -217,7 +217,8 @@ class Monitor : public Dispatcher,
   std::vector<DaemonHealthMetric> get_health_metrics();
 
   int quorum_age() const {
-    auto age = ceph::mono_clock::now() - quorum_since;
+    auto age = std::chrono::duration_cast<std::chrono::seconds>(
+      ceph::mono_clock::now() - quorum_since);
```

Comment:
This was in my original commit but the current code was actually suggested by @tchaikov:

https://github.com/ceph/ceph/pull/42924#discussion_r696664160

(in an obsolete PR)

The implication was, I thought, that the duration cast was unnecessary.

Reply:
I think the cast is necessary or else the return value is not seconds.. i think it is nanoseconds or something?  i'm basically getting a random signed integer for every ``ceph -s`` call.

Reply:
Works for me. I trust you :)